src/CodeCompliance/Domain/Checks/PrivateApi/Used/DependencyProvider.php

```diff
@@ -32,6 +33,14 @@ public function getGuideline(): string
         return 'Please avoid usage of %s::%s in %s';
     }
 
+    /**
+     * @return string
+     */
+    public function getDocumentationUrl(): string
+    {
+        return static::DOCUMENTATION_BASE_URL . 'private-api-is-extended.html';
```

Comment:
https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/upgradability-guidelines/private-api-is-extended.html#example-of-code-that-causes-an-upgradability-error-extending-a-private-api-dependency-provider